ubuntu/Dockerfile

```diff
@@ -1,8 +1,105 @@
 FROM amd64/ubuntu:20.04
+
+# docker-to-linux
 LABEL com.iximiuz-project="docker-to-linux"
 RUN apt-get update -y
 RUN apt-get -y install \
   linux-image-virtual \
   systemd-sysv
 RUN echo "root:root" | chpasswd
 
+# docker-remote-desktop
+RUN sed -i -E 's/^# deb-src /deb-src /g' /etc/apt/sources.list \
+    && apt-get update \
+    && DEBIAN_FRONTEND="noninteractive" apt-get install -y --no-install-recommends \
+        build-essential \
+        dpkg-dev \
+        git \
+        libpulse-dev \
+        pulseaudio \
+    && apt-get build-dep -y pulseaudio \
+    && apt-get source pulseaudio \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN cd /pulseaudio-$(pulseaudio --version | awk '{print $2}') \
+    && ./configure
+
+RUN git clone https://github.com/neutrinolabs/pulseaudio-module-xrdp.git /pulseaudio-module-xrdp \
+    && cd /pulseaudio-module-xrdp \
+    && ./bootstrap \
+    && ./configure PULSE_DIR=/pulseaudio-$(pulseaudio --version | awk '{print $2}') \
+    && make \
+    && make install
+
+RUN apt-get update \
+    && DEBIAN_FRONTEND="noninteractive" apt-get install -y --no-install-recommends \
+        dbus-x11 \
+        firefox \
+        git \
+        locales \
+        pavucontrol \
+        pulseaudio \
+        pulseaudio-utils \
+        sudo \
+        x11-xserver-utils \
+        xfce4 \
+        xfce4-goodies \
+        xfce4-pulseaudio-plugin \
+        xorgxrdp \
+        xrdp \
+        xubuntu-icon-theme \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN sed -i -E 's/^; autospawn =.*/autospawn = yes/' /etc/pulse/client.conf \
+    && [ -f /etc/pulse/client.conf.d/00-disable-autospawn.conf ] && sed -i -E 's/^(autospawn=.*)/# \1/' /etc/pulse/client.conf.d/00-disable-autospawn.conf || : \
+    && locale-gen en_US.UTF-8
+ENV LANG en_US.UTF-8
+
+# docker-wine
+# Install prerequisites
```

Comment:
copy and paste hackjob from https://github.com/scottyhardy/docker-wine/blob/master/Dockerfile